services/API-service/src/api/notification/whatsapp/whatsapp.service.ts

```diff
@@ -45,16 +45,26 @@ export class WhatsappService {
   ): Promise<void> {
     const validatedPhoneNumber =
       await this.lookupService.lookupAndCorrect(recipientPhoneNr);
-    await this.sendWhatsapp(message, validatedPhoneNumber);
+    await this.sendWhatsapp(message, null, null, validatedPhoneNumber);
   }
 
   private async sendWhatsapp(
     message: string,
+    contentSid: string,
+    contentVariables: object,
     recipientPhoneNr: string,
     mediaUrl?: string,
   ) {
     const payload = {
-      body: message,
+      body:
```

Comment:
NICE TO HAVE: Improve readability by reducing repetitive checks.

```typescript
const isProduction = process.env.NODE_ENV === 'production';

const payload = {
    body: isProduction && contentSid ? undefined : message,
    contentSid: isProduction ? contentSid : null,
    contentVariables: isProduction && contentVariables
        ? JSON.stringify(contentVariables)
        : undefined,
    messagingServiceSid: process.env.TWILIO_MESSAGING_SID,
    from: `whatsapp:${process.env.TWILIO_WHATSAPP_NUMBER}`,
    statusCallback: EXTERNAL_API.whatsAppStatus,
};
```

Of course, there are other approaches based on your preference.